CONTRIBUTING.md

```diff
@@ -219,12 +219,11 @@ Windows.
 that require an case sensitive filesystem (default in Linux, configurable in **MacOS/Windows**),
 make sure you run all this from a filesystem that supports this. Here you have [some instructions](https://github.com/openSUSE/open-build-service/wiki/Setup-an-OBS-Development-Environment-on-macOS) in case you are a MacOS user.
 
-1. Install [docker](https://www.docker.com) and [docker-compose (version >= 1.20.0)¹](https://docs.docker.com/compose/).
+1. Install [docker](https://www.docker.com)
```

Comment:
```suggestion
1. Install [docker (version >= v2)¹](https://www.docker.com)
```
We should not forget the reference described with the `¹`.

And also, shouldn't be the minimum required version  `v2`?

Reply:
Yes, it should be